site/docs/config.toml

```diff
@@ -50,6 +50,7 @@ ignoreFiles = [
   "/ci/",
   "/hw/vendor/*",
   "/sw/vendor/*",
+  "/sw/device/tock/tock/*",
```

Comment:
ignoreFiles is a (probably go-style) regex, not a glob pattern (https://gohugo.io/getting-started/configuration/#ignore-content-and-data-files-when-rendering). It's not clear what path is compared to the patterns here, so probably removing the `*` might do the trick, or make it `.*` for a regex-equivalent of a star.

Reply:
Using the `*` does work and is being used above for the hw/sw vendor directories.

I can try with `.*` though. I should change the other uses of `*` at the same time.

Reply:
Yeah, the list is a bit of a mixed bag. the '*' works because it is a valid regex, just not doing what the author intended.

Reply:
Oddly `.*` didn't work either. After a bit of experimentation it seems the path getting filtered was `/sw/device/tock/tock` and not `/sw/device/tock/tock/Cargo.toml` so using `/sw/device/tock/tock/*` works because it means it just ignores the final `/`but `/sw/device/tock/tock/.*` didn't work because it wasn't passing a path with the final `/` through the filter and apparently not the filename itself either.

The solution is to use `/sw/device/tock/.*`

I also added a commit to fix up some other `*` usages